technische-universitat-dresden-betriebswirtschaftslehre-rechnungswesen-controlling.csl

```diff
@@ -0,0 +1,1655 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
+  <info>
+    <title>Technische Universität Dresden - Betriebswirtschaftslehre/Rechnungwesen/Controlling (APA)</title>
```

Comment:
I'm also wondering if this should be renamed to "... - BWL, insb. Betriebliches Rechnungswesen/Controlling (APA)", if this style is for that department (I don't see any hits on Google for the verbatim "Betriebswirtschaftslehre/Rechnungwesen/Controlling").

Reply:
Fair enough. I was just being consistent with the other Dresden styles

Reply:
Ah, right, let's keep it like this, then.